init.lua

```diff
@@ -71,13 +71,34 @@ if minetest.settings:get_bool("random_messages_api.load_custom_messages", true)
     end
 end
 
+minetest.register_chatcommand("toggle_random_messages", {
+    params = "",
+    description = "Toggle whether to show the random messages to you",
```

Comment:
```suggestion
    description = S("Toggle whether to show the random messages to you"),
```

Of course, `local S = ...` has to be added to the top.